content/departments/engineering/dev/career-development/framework.md

```diff
@@ -401,23 +400,60 @@ To learn more, see ["Considerations for promotion" in our talent review process]
+        <div class="wrapper">
+          A respected technical leader, on and off their team.
+        </div>
+      </td>
+      <td class="category-summary execution">
+        <div class="wrapper">
+          Defines deep or broad goals that align with top level company goals, and ensures delivery to meet business needs.
+        </div>
+      </td>
+      <td class="category-summary teamwork">
+        <div class="wrapper">
+          An extremely strong communicator doing outstanding stakeholder management.
+        </div>
+      </td>
+    </tr>
+    <tr class="behaviors-row">
+      <td class="behaviors proficiency">
+        <ul>
+          <li>Sets the technical vision for their team, and influences the broader technical vision.</li>
+          <li>Initiates and drives projects with broad/deep impact that enable higher quality work.</li>
+          <li>Provides oversight, coaching, and guidance through code and design reviews, both on and off the team.</li>
+          <li>Acts as a trusted advisor, drawing on functional expertise to inform customer-driven strategy.</li>
+        </ul>
+      </td>
+      <td class="behaviors execution">
+        <ul>
+          <li>Proactively identifies areas for improvement across engineering. Suggests process and methodology improvements.</li>
+          <li>Works closely with Engineering/Product leadership to validate alignment of team roadmaps within their org.</li>
+          <li>Independently scopes, designs, and delivers solutions for large, complex challenges.</li>
+          <li>Operates with calm and grace in ambiguous/uncertain situations. Does not shy away from taking calculated risks.</li>
+        </ul>
+      </td>
+      <td class="behaviors teamwork">
+        <ul>
+          <li>Provides technical expertise internally and externally, informing what can be achieved.</li>
+          <li>Regularly shares knowledge to influence and up-level large and/or senior audiences.</li>
+          <li>Persuades and challenges clients and internal stakeholders, using valid expertise and respectful communication.</li>
+          <li>Responsive to a variety of unexpected requests for advice or consultation; gracefully handles more frequent context-switching.</li>
+        </ul>
       </td>
     </tr>
     <!-- IC6 -->
     <tr>
       <th id="ic6" scope="row" class="level"><a class="anchor" href="#ic6"></a><abbr title="Individual Contributor">IC</abbr>6</th>
       <td colspan="3" class="tbd">
         <p>
-          We haven't yet finalized the description of this level at Sourcegraph.
-          Like <a href="#ic5">IC5</a>, this is a different <strong>role</strong> than the levels preceding it based not only on performance, but also business need, and (like <a href="#ic5">IC5</a>) what impact at this level looks like may vary more from person to person than at preceding levels.
+          Senior Staff Engineer. We haven't yet finalized the description of this level at Sourcegraph.
```

Comment:
Is it Senior Staff Engineer? Or is it a Principle Engineer? (Or, is the answer we don't know that yet either, in which case I think it shouldn't be stated yet?)

Reply:
The typical leveling in Big Tech is Staff -> Sr. Staff -> Principal; maybe we can call it Sr. Staff+ (TBD) for now.

Reply:
This was proposed by Quinn [here](https://sourcegraph.slack.com/archives/C03KFJTH5TN/p1658800158412539?thread_ts=1658504132.726919&cid=C03KFJTH5TN), and restated [here](https://docs.google.com/document/d/16fL4KETgtPPbhux9DkZC6hLbJg6qQO8bw2REuqA8qXs/edit#heading=h.gwxh3351wx5i), and no one commented on it, so I took that to mean we were all happy with this title.